hummingbot/market/binance/binance_market.pyx

```diff
@@ -633,6 +633,10 @@ cdef class BinanceMarket(MarketBase):
                 client_order_id = tracked_order.client_order_id
                 if isinstance(order_update, Exception):
                     if order_update.code == 2013 or order_update.message == "Order does not exist.":
+                        self.logger().network(
+                            f"Order: {client_order_id} does not exist in Tracked orders :{order_update}."
```

Comment:
This shouldn't be network error. You can do a warning log instead. 
I think triggering the market order failure event is enough.